GCTC-NTGC/gc-digital-talent
# Hydrogen 2.0.0

## 👋 Introduction

Updates Hydrogen to a stable release, no breaking changes expected for a long while now :)

## 🕵️ Details

- There are a handful of changes here, mostly cosmetic key changes to the config file to make certain settings much clearer as to what they actually do.
- The other big change is that the `modes` setting now has a global `method` value, which prepares the system for a complimentary `:contrast` mode in the future. 👀 

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Check out the branch
2. Re-run the maintenance setup script to ensure you have the latest module
3. Run dev or storybook
4. Ensure that Hydrogen reports `2.0.0` being used in its output
5. Ensure there are no Hydrogen or compile errors
